jsonexamples/generated/genfeaturejson.rb

```diff
@@ -39,7 +39,7 @@ def write_half(filename, start1, repeat1, end1, repeat2)
         halfway_point = start1.bytesize + repeat1_len + repeat2.bytesize
 
         puts "Writing #{filename} ..."
-        File.open(filename, "w") do |file|
+        File.open(filename, "wb") do |file|
```

Comment:
I have not actually read this script, but why is the "b" necessary here? Is this non-text data?

Reply:
That prevents Ruby from writing \n as \r\n, which messes up the sizing.

Reply:
+1